Website/src/rendering/src/components/Checkout/CartDetails.tsx

```diff
@@ -17,7 +18,9 @@ const CartDetails = (props: CartDetailsProps): JSX.Element => (
         <div className="cart-details-actions-wrapper">
           <CartSummary />
           <PromoInput />
-          <button className="btn--main btn--main--round">Proceed to Checkout</button>
+          <button className="btn--main btn--main--round">
```

Comment:
The HTML language mentions that an anchor (interactive element) is not a permitted content in a button form element: https://developer.mozilla.org/en-US/docs/web/html/element/button

In other places, we have that construct:
```
<Link href="/tickets">
  <a className="btn--main btn--main--round">Book Tickets</a>
</Link>
```

When we will run code and call services, we could convert it as a submit button and handle the redirect in code only when all calls are succeeding.

Reply:
I knew that the button will have functionality, so I just inserted the link for the time being because there was a link to checkout in the cart page and I didn't want to lose that. I didn't think about the markup tbh :) I will convert it to an <a> tag.